storage/bucket.go

```diff
@@ -232,25 +233,21 @@ func (b *BucketHandle) newPatchCall(uattrs *BucketAttrsToUpdate) (*raw.BucketsPa
 // access to a restricted resource for a limited time without needing a
 // Google account or signing in. For more information about signed URLs, see
 // https://cloud.google.com/storage/docs/accesscontrol#signed_urls_query_string_authentication
+//
+// This method only requires the Method and Expires fields in the specified
+// SignedURLOptions opts to be non-nil. If not provided, it attempts to fill the
+// GoogleAccessID and PrivateKey from the GOOGLE_APPLICATION_CREDENTIALS environment variable.
+// If no private key is found, it attempts to use the GoogleAccessID to sign the URL.
+// This requires the IAM credentials API to be enabled and iam.serviceAccounts.signBlob
+// permissions on the service account.
+// If you do not want these fields set for you, you may use
+// SignedURL(bucket, name string, opts *SignedURLOptions) instead.
```

Comment:
Alternatively, they can still use this method and provide the auth options themself-- we only attempt to fill in if there is something missing, correct?

Reply:
Correct, I'll fix the comment to make that clearer.